crates/meilisearch/tests/tasks/mod.rs

```diff
@@ -127,93 +128,96 @@ async fn list_tasks_with_star_filters() {
 
     let (response, code) =
         index.service.get("/tasks?types=*,documentAdditionOrUpdate&statuses=*").await;
-    assert_eq!(code, 200, "{:?}", response);
+    assert_eq!(code, 200, "{response:?}");
     assert_eq!(response["results"].as_array().unwrap().len(), 2);
 
     let (response, code) = index
         .service
-        .get("/tasks?types=*,documentAdditionOrUpdate&statuses=*,failed&indexUids=test")
+        .get(format!(
+            "/tasks?types=*,documentAdditionOrUpdate&statuses=*,failed&indexUids={}",
+            index.uid
+        ))
         .await;
-    assert_eq!(code, 200, "{:?}", response);
+    assert_eq!(code, 200, "{response:?}");
     assert_eq!(response["results"].as_array().unwrap().len(), 2);
 
     let (response, code) = index
         .service
         .get("/tasks?types=*,documentAdditionOrUpdate&statuses=*,failed&indexUids=test,*")
         .await;
-    assert_eq!(code, 200, "{:?}", response);
+    assert_eq!(code, 200, "{response:?}");
     assert_eq!(response["results"].as_array().unwrap().len(), 2);
 }
 
 #[actix_rt::test]
 async fn list_tasks_status_filtered() {
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
```

Comment:
```suggestion
    // Do not use a shared server because we want to assert stuff against the global list of tasks
    let server = Server::new().await;
    let index = server.index("test");
```